tests/FSharp.Compiler.ComponentTests/TypeChecks/Graph/CompilationTests.fs

```diff
@@ -13,7 +13,7 @@ type Method =
 let methodOptions (method: Method) =
     match method with
     | Method.Sequential -> []
-    | Method.Graph -> [ "--test:GraphBasedChecking"; "--test:DumpCheckingGraph" ]
+    | Method.Graph -> [ "GraphBasedChecking"; "DumpCheckingGraph" ]
```

Comment:
Maybe keep `--test:DumpCheckingGraph` as test flag? It still very much is a contributor debug thing.

Reply:
Yeah, can't remove them due to backwards compatibility

Reply:
Done. Thanks